src/views/CreateTodo.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from "react";
+import InputBox from "../components/InputBox";
+import RadioInput from "../components/RadioInput";
+import Button from "../components/Button";
+
+export default class CreateTodo extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      responsible: "",
+      description: "",
+      priority: "",
+      isCompleted: false
+    };
+
+    this.onInputChangeHandler = this.onInputChangeHandler.bind(this);
```

Comment:
Change these functions into arrow functions. you don't need to bind these afterwords.

Reply:
Okay...